client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx

```diff
@@ -20,9 +20,9 @@ const Subscribers: Step = function ( { navigation } ) {
 		submit?.();
 	};
 
-	const isSiteOnFreePlan = !! site?.plan?.is_free;
+	const hasSubscriberLimit = !! site?.plan?.is_free;
```

Comment:
Should we include hosting trial check here as well?

Reply:
Does it make sense? Are we using the onboarding flow for anything that's not creating a site? Are hosting trialers going to see this step?